src/utils/flux/connect/__spec__.js

```diff
@@ -86,6 +86,15 @@ describe('connect', () => {
     expect(mapStateToProps).toHaveBeenCalledWith(testStore.getState(), wrapper.props());
   });
 
+  it('props from mapStateToProps take precedence', () => {
+    const Presenter = () => null;
+    const mapStateToProps = props => ({ text: `modified ${props.text}` });
+    const ConnectedComponent = connect(mapStateToProps)(Presenter);
+    const wrapper = mount(<ConnectedComponent text={ 'sample text' } />).find(Presenter);
```

Comment:
Have you see [`dive()`](http://airbnb.io/enzyme/docs/api/ShallowWrapper/dive.html) which I used to get round having to use `mount` when using HOCs (not sure if it would actually work here, but worth knowing about)

Reply:
I would have prefered to have done this but after shallow rendering then looking at the sub-component using `dive` didn't return the `Presenter ` component ¯\_(ツ)_/¯

Reply:
I've changed the test to use `shallow`, but it turns out we don't need to use `dive` here as shallow returns the return of the HOC, which will be the base component that is being wrapped.